### PR TITLE
[tests] remove/speedup some low signal tests

### PR DESCRIPTION
**Pull request by a-r-r-o-w**
# What does this PR do?

As discussed internally with @DN6.

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: tests/pipelines/test_pipelines_common.py [hunk 1]
```diff
@@ -290,7 +273,15 @@ def _modify_inputs_for_ip_adapter_test(self, inputs: Dict[str, Any]):
         inputs["return_dict"] = False
         return inputs
 
-    def test_ip_adapter_single(self, expected_max_diff: float = 1e-4, expected_pipe_slice=None):
+    def test_ip_adapter(self, expected_max_diff: float = 1e-4, expected_pipe_slice=None):
+        r"""Tests for IP-Adapter.
+
+        The following scenarios are test:
```
Comment (a-r-r-o-w) [inline tests/pipelines/test_pipelines_common.py:279]:
```suggestion
        The following scenarios are tested:
```

File: tests/pipelines/test_pipelines_common.py [hunk 2]
```diff
@@ -290,7 +273,15 @@ def _modify_inputs_for_ip_adapter_test(self, inputs: Dict[str, Any]):
...
+          - Single IP-Adapter with scale=0 should produce same output as no IP-Adapter.
+          - Multi IP-Adapter with scale=0 should produce same output as no IP-Adapter.
+          - Single IP-Adapter with scale!=0 should produce different output as no IP-Adapter.
+          - Multi IP-Adapter with scale!=0 should produce different output as no IP-Adapter.
```
Comment (a-r-r-o-w) [inline tests/pipelines/test_pipelines_common.py:283]:
```suggestion
          - Single IP-Adapter with scale!=0 should produce different output compared to no IP-Adapter.
          - Multi IP-Adapter with scale!=0 should produce different output compared to no IP-Adapter.
```

File: tests/pipelines/test_pipelines_common.py [hunk 3]
```diff
@@ -1689,28 +1672,6 @@ def _test_xformers_attention_forwardGenerator_pass(
         if test_mean_pixel_difference:
             assert_mean_pixel_difference(output_with_offload[0], output_without_offload[0])
 
-    def test_progress_bar(self):
```
Comment (DN6) [inline tests/pipelines/test_pipelines_common.py:1692]: Would actually test this once. We can add it here 
https://github.com/huggingface/diffusers/blob/main/tests/pipelines/test_pipeline_utils.py

**Comment by a-r-r-o-w** [comment]
cc @yiyixuxu since this modifies the IP adapter tests that we worked on together when they were first introduced. TLDR; combining the multi and single ip-adapter tests, and enable/disable freeu tests, saves us roughly 40-50 forward passes

**Review by DN6 — APPROVED** [review]

**Pull request merged**